lib/plugins/physics.js

```diff
@@ -14,7 +14,7 @@ const PI_2 = Math.PI * 2
 const PHYSICS_INTERVAL_MS = 50
 const PHYSICS_TIMESTEP = PHYSICS_INTERVAL_MS / 1000
 
-function inject (bot) {
+function inject (bot, { physicsEnabled }) {
```

Comment:
where does `physicsEnabled` get set?

Reply:
The second argument to a plugins' Inject function is always the options object passed to createBot, as called in https://github.com/PrismarineJS/mineflayer/blob/HEAD/lib/plugin_loader.js#L41 . By destructuring like this, we are only taking the physicsEnabled property from the options object.
See https://github.com/PrismarineJS/mineflayer/blob/master/lib/plugins/blocks.js#L23 for similar usage in an existing plugin